### ISSUE TITLE
TensorBoard Warning

### ISSUE THREAD

**Issue by SeaLu** [ultralytics/ultralytics#7714]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

![{0ZLW7H645`QQ(NE}ECH6AL](https://github.com/ultralytics/ultralytics/assets/78088515/c59ec267-ca69-4a83-ba22-c51662bc2231)
How to address this warning？

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@SeaLu thank you for the bug report, I'm able to reproduce this myself with:

```
!yolo train model=rtdetr-l.pt data=coco8.yaml epochs=3 imgsz=640
```

<img width="1502" alt="Screenshot 2024-01-21 at 00 21 26" src="https://github.com/ultralytics/ultralytics/assets/26833433/4f211a40-4d67-4190-849d-75ca54e7f1af">

**Comment by glenn-jocher** [comment]
@SeaLu should be resolved now in #7725

**Comment by glenn-jocher** [comment]
Result with PR:
<img width="2171" alt="Screenshot 2024-01-21 at 17 30 25" src="https://github.com/ultralytics/ultralytics/assets/26833433/9534b9d0-878a-4388-9b4d-952dad709471">
